tests/unit/spanner_dbapi/test_connection.py

```diff
@@ -157,9 +157,6 @@ def test_transaction_checkout(self):
         mock_transaction.committed = mock_transaction.rolled_back = False
         self.assertEqual(connection.transaction_checkout(), mock_transaction)
 
-        connection._autocommit = True
```

Comment:
If you use `run_in_transaction` for batch updates in autocommit mode, then you don't have to change this behavior.